lib/src/native/realm_core.dart

```diff
@@ -2332,6 +2332,11 @@ class _RealmCore {
       return out_did_run.value;
     });
   }
+
+  void writeCopy(Realm realm, Configuration config) {
+    final configHandle = _createConfig(config);
+    _realmLib.invokeGetBool(() => _realmLib.realm_convert_with_config(realm.handle._pointer, configHandle._pointer, false));
```

Comment:
`realm_convert_with_config` supports merging with existing file. Should we be exposing this functionality as well?

Reply:
It is not exposed by the other SDKs.